.github/workflows/linkcheck.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           sudo apt-get install pandoc
           python -m pip install --upgrade pip
           pip install numpy matplotlib scipy NEURON
-          pip install ipywidgets voila
+          pip install 'ipywidgets<=7.7.1' voila 'ipympl<0.9'
```

Comment:
if you change this in `setup.py` and then do `pip install .[gui]` you might have to change it in only one location?